src/includes/performance/retrieve-transaction/php.mdx

```diff
@@ -0,0 +1,54 @@
+## Setting and Retrieving the Current Transaction
+
+In cases where you want to attach spans to an already ongoing transaction you can use `getTransaction()` on the current hub. This property will return a `\Sentry\Tracing\Transaction` in case there is a transaction set otherwise it returns `null`.
```

Comment:
```suggestion
In cases where you want to attach spans to an already ongoing transaction, you can use `getTransaction()` on the current hub. This property will return a `\Sentry\Tracing\Transaction` in cases where there is a transaction set; otherwise, it returns `null`.
```